scripts/install_avalanchego_release.sh

```diff
@@ -27,20 +27,64 @@ if [[ ${GOOS} == "darwin" ]]; then
   AVAGO_DOWNLOAD_PATH=${BASEDIR}/avalanchego-macos-${VERSION}.zip
 fi
 
-AVALANCHEGO_BUILD_PATH=${AVALANCHEGO_BUILD_PATH:-${BASEDIR}/avalanchego-${VERSION}}
+# TODO: Please read:
+# not sure why the version isn't included in CI currently... open to suggestions on how to handle this better
+AVALANCHEGO_BUILD_PATH=${AVALANCHEGO_BUILD_PATH-${VERSION}:-${BASEDIR}/avalanchego-${VERSION}}
 mkdir -p $AVALANCHEGO_BUILD_PATH
 
 if [[ ! -f ${AVAGO_DOWNLOAD_PATH} ]]; then
   echo "downloading avalanchego ${VERSION} at ${AVAGO_DOWNLOAD_URL} to ${AVAGO_DOWNLOAD_PATH}"
```

Comment:
could we move this to the download part of the `if` or update the language so it says `checking for` instead of downloading?

Reply:
you bet

Reply:
![Screenshot 2023-03-22 at 3 21 00 PM](https://user-images.githubusercontent.com/3286504/227013972-aeb66196-73fe-46aa-a536-8dc9c2f975ad.png)

thoughts